var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -21,6 +21,7 @@ class Fms(CMakePackage):
     variant('gfs_phys', default=True, description='Use GFS Physics?')
     variant('openmp', default=True, description='Use OpenMP?')
 
+    version('2021.04', sha256='dcb4fe80cb3b7846f7cf89b812afff09a78a10261ea048a851f28935d6b241b1')
```

Comment:
Why is the format of the version number different from below? Should it be just `2021.04` or `2021.04.01`?